test/test_kernel_smooth_quantile_fn.py

```diff
@@ -1,3 +1,6 @@
+import sys
+import os
+sys.path.append(os.path.join(os.path.dirname(__file__), '..')) 
```

Comment:
I'm ok with leaving this `sys.path.append` line in the file if we need to, but I was able to run the unit tests before without it, so I suspect it is not necessary?